installation/routines/customize_options.sh

```diff
@@ -208,25 +208,38 @@ Would you like to install the Web App? [Y/n]"
 }
 
 _option_kiosk_mode() {
-  # ENABLE_KIOSK_MODE
-  clear_c
-  print_c "----------------------- KIOSK MODE ----------------------
+    # ENABLE_KIOSK_MODE
+    clear_c
+    print_c "----------------------- KIOSK MODE ----------------------"
+    if [[ $(get_architecture) == "armv6" ]]; then
+
+        print_c "
+Due to limited resources the kiosk mode is not supported
+on Raspberry Pi 1 or Zero 1 ('ARMv6' models).
+
+Press enter to continue."
+        read
```

Comment:
Maybe add also something like "Disabling kiosk mode". 

Just "Continue" sounds a little like it is enabled anyway.

Reply:
Changed the message to "will not be installed".
"disable" could suggest it was enabled before.